<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>183</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Search settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="fuzzySearchLabel">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enables the error tolerant search.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Fuzzy search</string>
        </property>
        <property name="buddy">
         <cstring>fuzzySearchCheckBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="fuzzySearchCheckBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="errorToleranceLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Determines the amount of allowed errors. Technically speaking this is the prefix edit distance. Allowed values are an absolute number of errors or a percentage of errors relative to the word length. The latter is given by a floating point number. Since the value is always smaller than one the leading zero is omitted. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Error tolerance</string>
        </property>
        <property name="buddy">
         <cstring>errorToleranceLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="errorToleranceLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="sizeOfQGramsLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Size of the grams. Refer to the wikipedia artile for a detailed explanation of this value. Three is a reasonable value for this.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Size of qGrams </string>
        </property>
        <property name="buddy">
         <cstring>sizeOfQGramsSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sizeOfQGramsSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fuzzySearchCheckBox</tabstop>
  <tabstop>errorToleranceLineEdit</tabstop>
  <tabstop>sizeOfQGramsSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>fuzzySearchCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>errorToleranceLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fuzzySearchCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>errorToleranceLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fuzzySearchCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>sizeOfQGramsLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fuzzySearchCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>sizeOfQGramsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
